### PR TITLE
Update README with new `deviceTypes` from my system

### PR DESCRIPTION
**Pull request by kevin-david**
Also add basic prereq install instructions which I discovered by debugging.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,18 +23,23 @@ Or clone the repo:
...
-| 19         | Glass Break Detector |
+| deviceType | Description                 |
+|------------|-----------------------------|
+| 1          | Contact Sensor              |
+| 2          | Motion Sensor               |
+| 5          | Smoke Detector              |
+| 6          | CO Detector                 |
+| 9          | Panic Button                |
+| 19         | Glass Break Detector        |
+| 68         | Panel(?) Image Sensor       |
```
Comment (kevin-david) [inline README.md:41]: Not sure on this one. On my system, panel motion detector != motion sensor, so I _bet_ it's different but not 100% positive.

**Comment by elahd** [comment]
@kevin-david Would you be able to update the code as well so that the library supports the new sensors?

**Comment by kevin-david** [comment]
@elahd Not sure I can pull images but I can't imagine getting panel motion working would be hard. I need to rebase this after the 0.2.0 changes. Maybe I can take a look this weekend!

**Comment by kevin-david** [comment]
@elahd  updated to add a new `PANEL_MOTION_SENSOR` enum value. I couldn't see how `MOTION_SENSOR` was consumed anywhere in the integration... does it happen auto-magically somehow?

**Comment by elahd** [comment]
Awesome! The integration picks up any sensors that are defined in pyalarmdotcomajax's `ADCSensorSubtype` enum, so your changes are enough. The integration will create a generic binary sensor by default. You can have these sensors appear as motion sensors by changing [`alarmdotcom.binary_sensor.device_class`](https://github.com/uvjustin/alarmdotcom/blob/21c737bcf7ec52545ea9d4f7099c68f38f99453c/custom_components/alarmdotcom/binary_sensor.py#L99) to set `device_class` for `ADCSensorSubtype.PANEL_MOTION_SENSOR` and `ADCSensorSubtype.MOTION_SENSOR` to `bsdc.MOTION`.

**Pull request merged**